src/assertions/assertions.csproj

```diff
@@ -10,17 +10,12 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="FluentAssertions" Version="6.8.0" />
-        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
+        <PackageReference Include="FluentAssertions" Version="6.12.0" />
+        <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All" />
     </ItemGroup>
 
     <ItemGroup>
         <ProjectReference Include="..\bctklib\bctklib.csproj" />
     </ItemGroup>
 
-
-    <ItemGroup>
-        <PackageReference Include="Neo" Version="$(NeoVersion)" />
-    </ItemGroup>
-
```

Comment:
Still need to `ref` the `submodule` of `neo`.

Reply:
`assertions` references `bctklib`, and `bctklib` references the `neo`